File: tests/Generator/resource/swift/complex/TypeAPI.swift
// The TypeAPI Root
class TypeAPI: TypeSchema {
    var baseUrl: String?
    var operations: Dictionary<String, Operation>?
    var security: Security?

    enum CodingKeys: String, CodingKey {
        case baseUrl = "baseUrl"
        case operations = "operations"
        case security = "security"
    }
}

File: tests/Generator/resource/csharp/complex/TypeAPI.cs
using System.Text.Json.Serialization;

namespace TypeAPI.Model;

/// <summary>
/// The TypeAPI Root
/// </summary>
public class TypeAPI : TypeSchema
{
    [JsonPropertyName("baseUrl")]
    public string? BaseUrl { get; set; }

    [JsonPropertyName("security")]
    public Security? Security { get; set; }

    [JsonPropertyName("operations")]
    public System.Collections.Generic.Dictionary<string, Operation>? Operations { get; set; }

}

